tests/server/flow-event.js

```diff
@@ -13,11 +13,9 @@ registerSuite('flow-event', {
   beforeEach: function() {
     config = {
       /*eslint-disable camelcase*/
-      client_metrics: {
-        stderr_collector_disabled: false
-      },
       flow_id_expiry: 7200000,
-      flow_id_key: 'foo'
+      flow_id_key: 'foo',
+      flow_metrics_disabled: false
```

Comment:
We are going to need a test if `flow_metrics_disabled` is true, then no flow events should be logged.